plugins/needs-review/webMonitor/trunk/WWW/_sidebar.html.template

```diff
@@ -2,18 +2,111 @@
 	<h5 class="center">
 		<i class="icon-map-marker"></i> <span class="value_field_description">{characterLocationDescription}</span>
 	</h5>
+    
 	<div class="center">
 		<!-- why canvas? -->
-		<canvas id="map" style="background-image: url('{characterLocationMapURL}')">
+        <!-- {characterLocationMapURL} -->
+		<canvas id="map" width="512" height="512" style="background-image: url('{characterLocationMapURL}'); width: 512px; height: 512px;">
 			<form action="/handler" method="GET">
 				<input type="hidden" name="csrf" value="{csrf}"/>
 				<input type="image" alt="{characterLocationMap}" src="{characterLocationMapURL}"/>
 			</form>
 		</canvas>
 	</div>
+    
 	<p>
 		<div class="center" id="coord">
 			<span class="value_char_pos_x">{characterLocationX}</span>, <span class="value_char_pos_y">{characterLocationY}</span>
 		</div>
 	</p>
+    
+    <script type="text/javascript">
+        $(function(){
+            var $map = $("#map");
+            
+            $map.on("click", function(){
+                var newWH = {
+                    width: "",
+                    height: "",
+                };
+                
+                if ($map.css("width") != "512px") {
+                    newWH = {
+                       width: "512px", 
+                       height: "512px", 
+                    };
```

Comment:
Yeah, currently i stuck with conversion of original coorditaions to map files. Annyn suggestions?

P.S. My local version works with party as well :)